Discovery doc on downtime for MHV

dcloud opened the pull request:
Write up on downtime notifications. How it's done currently, what MHV might need, future enhancements, etc.

- department-of-veterans-affairs/va.gov-team/issues/70798

dcloud committed:
Draft discovery doc on downtime for MHV

dcloud committed:
Add current priorities section, refactor enhancement

dcloud committed:
Add potential for MHV list of services enhancement. Edits to formatting,
etc.

dcloud committed:
Update notes about custom notification rendering

dcloud committed:
Add note about downtimeApproaching vs downtime.

carlosfelix2 commented on products/health-care/digital-health-modernization/engineering/mhv-downtime-notification-discovery.md:
```diff
@@ -0,0 +1,52 @@
+# MHV on VA.gov downtime notifications
+
+Created: 2023-12-21
```
This document read more like notes. Perhaps add a short introduction describing the purpose of this document. I read it and was not sure what to do with it.

wesrowe replied:
I feel the same way as Carlos. This doc reads as notes that could fuel a discussion of recommendations. It's a useful input to that work.

For example: below it's noted that the Appts team has used a `render` prop to customize the UX of the downtimeNotification component, rather than used it out-of-the-box. I'm glad you found that out. But now our team needs to understand the UX of that experience, and take it into account in our UX recommendation. Can we get screenshots?

dcloud replied:
Yes, they are notes from downtime discovery! I can add an intro saying that and adding some recommendations. Was hoping to generate discussion based on what's there, but it sounds like a read out might generate more useful discussion.

@wesrowe I don't have screenshots of the Appointments and don't know about getting them, but will look into it

dcloud replied:
Added a document purpose statement and a *Recommendations* section. And I think those may answer https://github.com/department-of-veterans-affairs/va.gov-team/issues/70798#issuecomment-1876136559 but please let me know what is still confusing

dcloud replied:
@wesrowe Added screenshots from Appointments/VAOS

dcloud committed:
Update mhv-downtime-notification-discovery.md

Add "Recommendations" section, clarify that this is a discovery doc.

dcloud committed:
Update mhv-downtime-notification-discovery.md

Added note about MaintenanceBanner

dcloud committed:
Update mhv-downtime-notification-discovery.md

Add VAOS screenshots

dcloud committed:
Update mhv-downtime-notification-discovery.md

Add note about header in `Down` component, clarify that `mhv` key is for auth downtime only

dcloud committed:
Update mhv-downtime-notification-discovery.md

Link to "Remove end time" PR https://github.com/department-of-veterans-affairs/vets-website/pull/10416

dcloud committed:
Update mhv-downtime-notification-discovery.md

Updates per discussion: https://github.com/department-of-veterans-affairs/va.gov-team/issues/70798#issuecomment-1881911322

dcloud commented:
Merged so folks can work on this doc directly on the main branch
